src/components/PageBlocks/RecordListConfigurator.vue

```diff
@@ -246,20 +246,13 @@ export default {
       }
     },
 
-    // Tries to determine ID of the page we're supposed to redirect
-    recordPageID () {
+    recordListModuleRecordPage () {
       // Relying on pages having unique moduleID,
-      const { moduleID } = this.recordListModule || {}
-      if (!moduleID) {
-        return undefined
-      }
-
-      const { pageID } = this.pages.find(p => p.moduleID === moduleID) || {}
-      if (!pageID) {
+      if (this.options.moduleID !== '0') {
```

Comment:
Can we start using `NoID` from `@cortezaproject/corteza-js` package? This applies to all of the below bits as well.

Reply:
Will change all '0' to NoID